app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainViewModel.kt

```diff
@@ -24,6 +28,14 @@ class MainViewModel : ViewModel() {
     fun getAllGroups(): ArrayList<Group> {
         groups = groupService.getAllGroups()
         return groups
-
     }
+
+
+    internal var notification: Notification
+    get() {return _notification}
+    set(value) {_notification = value}
+
+    internal var notifications: MutableLiveData<ArrayList<Notification>>
+    get() {return _notifications}
+    set(value) {_notifications}
```

Comment:
Should `_notifications` be set to `value`?

Reply:
Fixed, thanks